submissions/drewzag/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,61 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/drewzag/a-tiny-JS-world
+   Web app: https://drewzag.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+const dog = {
+  species: 'dog',
+  name: 'Jack',
+  gender: 'male',
+  legs: 4,
+  hands: 0,
+  saying: 'bark-bark!',
+}
+
+const cat = {
+  species: 'cat',
+  name: 'Tom',
+  gender: 'male',
+  legs: 4,
+  hands: 0,
+  saying: 'meow',
+}
+
+const woman = {
+  species: 'human',
+  name: 'Marry',
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  saying: 'Hi, guys)',
+}
+
+const man = {
+  species: 'human',
+  name: 'Boris',
+  gender: 'male',
+  legs: 2,
+  hands: 2,
+  saying: 'Dobriy den, everybody!',
+}
+
+const catWoman = {
+  species: 'cat-human',
+  name: 'Cat Marry',
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  saying: cat.saying,
```

Comment:
If we change `cat.saying`, would `catWoman.saying` change too? The task requires it to do so.

Reply:
honestly, I think yes, it would change)

Reply:
Oh, sorry) I totally forget about getter and setter